serverless.yml

```diff
@@ -37,6 +41,8 @@ functions:
       DB_NAME: ${self:custom.db.connectInfo.DATABASE_NAME}
       DB_PASSWORD: ${self:custom.db.connectInfo.SCHEMA_OWNER_PASSWORD}
       AWS_DEPLOYMENT_REGION: ${self:provider.region}
+      BUCKET_NAME: iow-retriever-capture-${self:custom.environments.${self:provider.stage}}
+      S3_OBJECT_SIZE_LIMIT: 150000000
```

Comment:
It's actually the trigger that currently has the limit control: https://github.com/usgs/aqts-capture-trigger/blob/master/serverless.yml#L49.

Reply:
Unless it's changed in the trigger, those large files won't make it to this lambda.